### DISCUSSION TITLE
Q: should variable `$EDITOR` used in default config expands in yazi UI?

### DISCUSSION THREAD

**Discussion by VKondakoff** [sxyazi/yazi#1219] [1. Q&A] [answered]
Should this variable (used in default yazi config) expand in UI? I was expecting to see the content of the $EDITOR instead of just variable name in the pop-up...

<img width="574" alt="Снимок экрана 2024-06-26 в 13 44 58" src="https://github.com/sxyazi/yazi/assets/230824/7c101d8e-dcf1-48fa-8ed4-2bbd76501a30">

**Comment by sxyazi** [comment, marked as answer]
I want to keep it, originally, IIRC its name was "Edit", but later it was changed to `$EDITOR` because many users reported to me that they couldn't edit files. 

I found that changing it to `$EDITOR` improved the situation a lot. When users see `$EDITOR`, they might wonder if they haven't set it themselves, or at least open the default configuration file to see what `$EDITOR` is. :/

BTW I also added a FAQ for it haha, https://yazi-rs.github.io/docs/faq#why-cant-edit

**Reply by VKondakoff** [reply]
Makes sense. Thanks for explanation!
